doc/man3/EVP_KDF.pod

```diff
@@ -208,6 +209,7 @@ If this memory usage limit is exceeded because the load factors are chosen
 too high, the key derivation will fail.
 
 The default value is implementation dependent.
+The memory size must never exceed what can be given with a B<size_t>.
```

Comment:
Is this enforced by the code?

Reply:
It's enforced enough by the way it's used